x/posts/handle_tx.go

```diff
@@ -0,0 +1,160 @@
+		return err
+	}
+
+	// Update the post attachments
+	err = m.updatePostAttachments(tx.Height, subspaceID, postID)
+	if err != nil {
+		return err
+	}
+
+	// Save the related transaction
+	return m.db.SavePostTx(types.NewPostTransaction(subspaceID, postID, tx.TxHash))
+}
+
+// parseEditPostEvent handles the edition of an existing post
+func (m *Module) parseEditPostEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := events.GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	postID, err := GetPostIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	// Update the post
+	err = m.updatePost(tx.Height, subspaceID, postID)
+	if err != nil {
+		return err
+	}
+
+	// Save the related transaction
+	return m.db.SavePostTx(types.NewPostTransaction(subspaceID, postID, tx.TxHash))
+}
+
+// parseDeletePostEvent handles the deletion of an existing post
+func (m *Module) parseDeletePostEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := events.GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	postID, err := GetPostIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	return m.db.DeletePost(tx.Height, subspaceID, postID)
+}
+
+// parseAddPostReactionEvent handles the addition of a reaction to an existing post
+func (m *Module) parseAddPostAttachmentEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := events.GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	postID, err := GetPostIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	// Update the attachments
+	err = m.updatePostAttachments(tx.Height, subspaceID, postID)
+	if err != nil {
+		return err
+	}
+
+	// Store the related post transaction
+	return m.db.SavePostTx(types.NewPostTransaction(subspaceID, postID, tx.TxHash))
+}
+
+// parseRemovePostAttachmentEvent handles the removal of a reaction from an existing post
+func (m *Module) parseRemovePostAttachmentEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := events.GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	postID, err := GetPostIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	attachmentID, err := GetAttachmentIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	// Delete the attachment
+	err = m.db.DeletePostAttachment(tx.Height, subspaceID, postID, attachmentID)
+	if err != nil {
+		return err
+	}
+
+	// Store the related post transaction
+	return m.db.SavePostTx(types.NewPostTransaction(subspaceID, postID, tx.TxHash))
+}
+
+// parseAnswerPollEvent handles the answer to a poll
+func (m *Module) parseAnswerPollEvent(tx *juno.Tx, event abci.Event) error {
+	subspaceID, err := events.GetSubspaceIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	postID, err := GetPostIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	pollID, err := GetPollIDFromEvent(event)
+	if err != nil {
+		return err
+	}
+
+	return m.updatePollAnswers(tx.Height, subspaceID, postID, pollID)
```

Comment:
I think it would be better to add `answer_indexes` and `answerer` attribute to Desmos core in the future so it can be updated directly rather than updating all the poll answers. What do you think?